SQLQuery_3(homework).sql

```diff
@@ -0,0 +1,303 @@
+CREATE DATABASE ZooManagement;
+
+USE ZooManagement;
+
+CREATE TABLE ZooInstitutions (
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    Name VARCHAR(100) NOT NULL, 
+    City VARCHAR(150) DEFAULT 'Country Romania',
+);
+
+
+CREATE TABLE Visitors (
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    TypeOfDay VARCHAR(50) NOT NULL,
+    FirstName VARCHAR(100) NOT NULL, 
+    LastName VARCHAR(100) NOT NULL,
+    ZooId INT REFERENCES ZooInstitutions(Id)
+
+);
+
+
+CREATE TABLE Tickets (
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    TypeOfTicket VARCHAR (100) NOT NULL,
```

Comment:
For TypeOfTicket a new table, maybe called, ValidityTypes could be used. Inside ValidityTypes we can store information like "one day", "one month", "one week" etc. and then it can be referenced in the Tickets table.